docs/book/provider_implementations/customize_resources.md

```diff
@@ -0,0 +1,56 @@
+# Customize Resources
+
+## Customize Cluster Role
```

Comment:
This shouldn't be needed as long as the proper annotations are added to the Actuators. Here is how we are doing it in cluster-api-provider-aws currently: https://github.com/kubernetes-sigs/cluster-api-provider-aws/blob/master/pkg/cloud/aws/actuators/machine/actuator.go#L45-L47

Reply:
I see. It's much simpler as it doesn't require the patch. I will make those changes and update the PR description accordingly.

Reply:
@detiber I see also you have the amazon provider's own objects in the rbac annotation:

https://github.com/kubernetes-sigs/cluster-api-provider-aws/blob/878bae8e17c8cfd38400511f3a542eafc4d50eae/pkg/cloud/aws/actuators/machine/actuator.go#L44

Should we also include the resources for the solas provider in the guide? I guess yes.

Reply:
No, those are not actually needed because they are not actually persisted as independent objects by the apiserver.